src/edu/ucsb/cs56/projects/games/roguelike/Player.java

```diff
@@ -17,12 +17,13 @@
     private int score;
     private String typeOfPiece;
     private char icon;
+    final static public int MAXHP = 100;// this is NEW
```

Comment:
Let's remove the comment.  It makes sense as a PR comment, but will be confusing it left in the code.